notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<p>Linear interpolation is inappropriate -- you already identified that there's a seasonal component that this will not respect. If you want to interpolate on a model like this you should use a statespace model and fill missing values with output from kalman smoother.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>